tests/utils/playwright/datalens/datalens-test-setup.ts

```diff
@@ -5,19 +5,17 @@ import {FullConfig, Page, chromium} from '@playwright/test';
 
 import {AuthRobotSettings} from '../types';
 
-import {authenticate} from './utils';
```

Comment:
Should we remove the authenticate from utils as well?

Reply:
upd: no, other tests are depends on that.

Reply:
I would only suggest creating a separate playwright config for public tests